### PR TITLE
Added supressions for v2 and supressed some old metrics.

### PR DESCRIPTION
**Pull request by maierlars**
### Scope & Purpose

Slightly improve documentation for a few metrics.
I ran the generateAllMetrics script and added the generated file to the PR.

Some metrics should be removed from v2. I added a `v2suppressions` map that filters those out.

- [x] :hankey: Bugfix (requires CHANGELOG entry)
- [ ] :pizza: New feature (requires CHANGELOG entry, feature documentation and release notes)
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification
- [x] :book: CHANGELOG entry made

### CODE REVIEW

**Comment by maierlars** [comment]
https://jenkins.arangodb.biz/job/arangodb-matrix-pr/14484/

File: Documentation/Metrics/arangodb_maintenance_action_done_total.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 unit: ms
 type: counter
 category: Maintenance
-complexity: advanced
+complexity: simple
 exposedBy:
   - dbserver
 description: |
-  Accumulated action runtime for Maintenance.
+  Counter of actions that are done and have been removed from the registry.
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_maintenance_action_done_total.yaml:11]: I think this should be a bit more elaborate, in particular if the complexity is simple. We should spend a few sentences on explaining what the maintenance is, what the expected values are and what to do if the values get out of hand.

File: Documentation/Metrics/arangodb_maintenance_action_duplicate_total.yaml [hunk 2]
```diff
@@ -4,7 +4,7 @@ help: |
 unit: number
 type: counter
 category: Maintenance
-complexity: medium
+complexity: advanced
 exposedBy:
   - dbserver
 description: |
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_maintenance_action_duplicate_total.yaml:10]: I think this description needs a bit more details.

File: Documentation/Metrics/arangodb_maintenance_action_failure_total.yaml [hunk 3]
```diff
@@ -4,7 +4,7 @@ help: |
 unit: number
 type: counter
 category: Maintenance
-complexity: medium
+complexity: simple
 exposedBy:
   - dbserver
 description: |
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_maintenance_action_failure_total.yaml:10]: same here

File: Documentation/Metrics/arangodb_maintenance_action_queue_time_msec.yaml [hunk 4]
```diff
@@ -4,8 +4,8 @@ help: |
 unit: ms
 type: histogram
 category: Maintenance
-complexity: medium
+complexity: advanced
 exposedBy:
   - dbserver
 description: |
-  Time spent in the queue before execution for maintenance actions.
+  Histogram of time spent in the queue before execution for maintenance actions.
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_maintenance_action_queue_time_msec.yaml:11]: same here.

File: Documentation/Metrics/arangodb_maintenance_action_registered_total.yaml [hunk 5]
```diff
@@ -4,7 +4,7 @@ help: |
 unit: number
 type: counter
 category: Maintenance
-complexity: medium
+complexity: simple
 exposedBy:
   - dbserver
 description: |
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_maintenance_action_registered_total.yaml:10]: same here.

File: Documentation/Metrics/arangodb_maintenance_action_runtime_msec.yaml [hunk 6]
```diff
@@ -4,7 +4,7 @@ help: |
 unit: ms
 type: histogram
 category: Maintenance
-complexity: medium
+complexity: advanced
 exposedBy:
   - dbserver
 description: |
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_maintenance_action_runtime_msec.yaml:10]: Same here.

File: Documentation/Metrics/arangodb_maintenance_agency_sync_runtime_msec.yaml [hunk 7]
```diff
@@ -4,8 +4,9 @@ help: |
 unit: ms
 type: histogram
 category: Maintenance
-complexity: medium
+complexity: simple
 exposedBy:
   - dbserver
 description: |
-  Histogram for total time spent on agency sync.
+  Histogram for total time spent on agency sync. This includes phase one and
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_maintenance_agency_sync_runtime_msec.yaml:11]: Same here.

File: Documentation/Metrics/arangodb_maintenance_phase1_runtime_msec.yaml [hunk 8]
```diff
@@ -4,7 +4,7 @@ help: |
 unit: ms
 type: histogram
 category: Maintenance
-complexity: simple
+complexity: advanced
 exposedBy:
   - dbserver
 description: |
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_maintenance_phase1_runtime_msec.yaml:10]: same here, probably need to explain about phase 1 and phase 2.

File: Documentation/Metrics/arangodb_maintenance_phase2_runtime_msec.yaml [hunk 9]
```diff
@@ -4,7 +4,7 @@ help: |
 unit: ms
 type: histogram
 category: Maintenance
-complexity: simple
+complexity: advanced
 exposedBy:
   - dbserver
 description: |
```
Comment (neunhoef) [inline Documentation/Metrics/arangodb_maintenance_phase2_runtime_msec.yaml:10]: same here.

**Review by neunhoef — CHANGES_REQUESTED** [review]
Some more details are missing. I will add the code to enact the suppressions.

**Review by neunhoef — APPROVED** [review]
LGTM

**Pull request merged**